packages/x-date-pickers-pro/src/DateRangePicker/date-range-manager.ts

```diff
@@ -6,27 +6,37 @@ interface CalculateRangeChangeOptions<TDate> {
   range: DateRange<TDate>;
   newDate: TDate;
   currentlySelectingRangeEnd: 'start' | 'end';
+  allowRangeFlip?: boolean;
```

Comment:
Could you document this param ? It's goal is not obvious from first read even if I think I get why it has been added

Reply:
Added JsDoc